haskell-lsp-types/src/Language/Haskell/LSP/Types/Message.hs

```diff
@@ -83,7 +83,7 @@ data ClientMethod =
  | WorkspaceSymbol
  | WorkspaceExecuteCommand
  -- Progress
- | WindowProgressCancel
+ | WorkDoneProgressCancel
```

Comment:
Does `window/workDoneProgress/cancel` exist anymore in the 3.15 specification? Are we supposed to just use https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#cancelRequest now?

Reply:
It still exists in https://github.com/microsoft/vscode-languageserver-node/blob/master/protocol/src/protocol.progress.proposed.ts#L127 so I assume that the omission from the spec was a mistake (given that other things like `workDoneProgress/create` are also not documented properly but I don’t know tbh.